### PR TITLE
Convert remaining Oracle curses to counters

### PR DESCRIPTION
**Pull request by TikaelSol**
Need help with this one because nothing is working properly.

### CODE REVIEW

File: packs/feat-effects/effect-curse-of-creeping-ashes.json [hunk 1]
```diff
@@ -101,35 +58,51 @@
...
                 "onDeleteActions": {
                     "grantee": "restrict"
                 },
                 "predicate": [
                     {
-                        "or": [
-                            "self:oracular-curse:stage:moderate",
-                            "self:oracular-curse:stage:major",
-                            "self:oracular-curse:stage:extreme"
+                        "gt2": [
```
Comment (stwlam) [inline packs/feat-effects/effect-curse-of-creeping-ashes.json:73]: gt or gte?

**Comment by simonward** [comment]
Fixes https://github.com/foundryvtt/pf2e/issues/8671

File: packs/feat-effects/effect-meddling-ancestor.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
+{
+    "_id": "vkgmMBMd2e2BwiwJ",
+    "img": "systems/pf2e/icons/spells/chromatic-image.webp",
+    "name": "Effect: Meddling Ancestor",
```
Comment (stwlam) [inline packs/feat-effects/effect-meddling-ancestor.json:4]: Can we use the nomenclature of the class feature here? "Ancestral Influence" as the name of the effect, and "Martial" as the "strike" influence

File: packs/feat-effects/effect-meddling-ancestor.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    "system": {
+        "badge": {
+            "evaluate": true,
+            "labels": [
+                "Battle",
```
Comment (TikaelSol) [inline packs/feat-effects/effect-meddling-ancestor.json:9]:
```suggestion
                "Martial",
```

File: packs/feat-effects/effect-meddling-ancestor.json (hunk 2 above)
Comment (TikaelSol) [reply packs/feat-effects/effect-meddling-ancestor.json:4]: I did use the nomenclature of the book for the effect name `Curse of Ancestral Meddling`, but we can change it easily. The label was just me going too quick and forgetting what they called it.

File: packs/feat-effects/effect-meddling-ancestor.json [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "label": "PF2E.OracleCurses.Ancestor.MeddlingAncestor",
```
Comment (stwlam) [inline packs/feat-effects/effect-meddling-ancestor.json:34]:
```suggestion
                "label": "PF2E.OracleCurses.Ancestor.AncestralInfluence",
```

File: packs/feat-effects/effect-meddling-ancestor.json [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+                "option": "meddling-ancestor",
+                "suboptions": [
+                    {
+                        "label": "PF2E.OracleCurses.Ancestor.Label.Battle",
```
Comment (stwlam) [inline packs/feat-effects/effect-meddling-ancestor.json:38]:
```suggestion
                        "label": "PF2E.OracleCurses.Ancestor.Label.Martial",
```

File: static/lang/re-en.json [hunk 6]
```diff
@@ -697,6 +697,7 @@
         "OracleCurses": {
             "Ancestor": {
                 "Label": {
+                    "Battle": "Battle",
```
Comment (stwlam) [inline static/lang/re-en.json:700]:
```suggestion
                    "Martial": "Martial",
```

**Pull request merged**